nodes/config.ts

```diff
@@ -11,8 +11,18 @@ function required(value: string | undefined): string {
 	return value;
 }
 
-export const config: Config = {
+const isDev = process.env.NODE_ENV === 'development';
+
+const devConfig: Config = {
 	formxApiBaseUrl: required(process.env.FORMX_API_BASE_URL),
 	formxWorkerBaseUrl: required(process.env.FORMX_WORKER_BASE_URL),
 	formxPortalBaseUrl: required(process.env.FORMX_PORTAL_BASE_URL),
 };
+
+const defaultConfig: Config = {
```

Comment:
I'd expect we have a config.ts for each env...?

Reply:
As this is a package to be published, I think we don't expect user to set env variable by themselves? And we should use formx's production environment by default?

Reply:
I think the issue is we are using this single config for both published package config and docker-based usage config.

Suggestion:
1. Ignore config.ts in git
2. Have two set of templates. One for docker (on pandawork), which uses env var, so there is only 1 template in this set. And then there is another set of templates, which uses hard coded value, so these would have config.production.ts, config.staging.ts etc. I think local should use the env var one? As its hosted on an n8n image via docker compose
3. In `make publish` (or the correct make target to publush it to npm), according to ENV set, copy the right template and rename to config.ts
4. In `make deploy` for deploying to pandawork, copy the env var template and rename to config.ts

Reply:
Good, just wanted to make it easier so didn't think deep about it

> In make publish (or the correct make target to publush it to npm), according to ENV set, copy the right template and rename to config.ts

But for publishing a public package, I think we always use the production config. So I would create `config.development.ts` and `config.production.ts` that development one is reading env var and production one is hardcoding value